.github/workflows/create-block.yml

```diff
@@ -0,0 +1,28 @@
+
+
+name: Create Block
+
+on: [push]
```

Comment:
We could [limit the workflow](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-paths) to only run on changes of specific packages instead of running it on all pushes.

Reply:
Nice feature, thank you sharing it. The only question is whether we put the filter on `packages` folder or we list all individual packages that might cause regressions. The latter would be ideal but it might be difficult to keep the list up to date.

Reply:
It didn't trigger an action with 24bdb32a4525e6057ca27d5455c002d7a76b506c so it seems to work correctly :)

Reply:
4a3e7f0 - `*.md` change only and nothing triggers, still good 👍